tests/test-ssh-user.cmd

```diff
@@ -0,0 +1,4 @@
+../footloose config create --config %t.footloose --name %t --key %t-key
```

Comment:
the tests expert `footloose` to be in the path, don't think we need to change that and so we should remove the `../`. Maybe using `go install` would help here. Something like:

```
go install # in the root of this repository
export PATH=`go env GOPATH`/bin:$PATH
```